### DISCUSSION TITLE
通过Github Action给lite分支自动打包

### DISCUSSION THREAD

**Discussion by moomiji** [hooke007/mpv_PlayKit#278] [mpv 通用]
目前的lite分支很适合合并本地的修改，像lua脚本、vs插件、着色器啥的可以用脚本从上游/main分支获取最新文件。

于是小摸了个bash脚本，借助Github Action自动打包客制化版本的MPV_lazy。

自动打包完成后，下载zip文档覆盖更新即可（前提是lite分支合并了客制化的修改）。

说明文档：https://github.com/moomiji/MPV_lazy/tree/lite/action
lite分支的action运行结果示例：https://github.com/moomiji/MPV_lazy/actions?query=branch%3Alite

pack.ini中已经预先根据`mpv-lazy-2023V2`版本，编写好了标准版的追新配置（VapourSynth为R61版本）

有需要的阅读说明文档食用，注意下mine分支带有我本地的修改，别合并错分支了。

**Comment by hooke007** [comment]
有两字体是0字节

**Reply by moomiji** [reply]
这我知道，原本是让留意到的人自行编写pack.ini的（
既然提出来了，已将字体的配置一起写入pack.ini

**Reply by hooke007** [reply]
肯定有人没看懂 https://github.com/hooke007/MPV_lazy/pull/279

**Reply by moomiji** [reply]
> 肯定有人没看懂 https://github.com/hooke007/MPV_lazy/pull/279

![](https://raw.githubusercontent.com/moomiji/MPV_lazy/lite/Temp/action/pr2.jpg)
草，f12改了下说明
不过github的pr默认是提交主仓库的，没用过的用户还是容易点错

**Comment by zhongfly** [comment]
我一开始以为是能让自己的配置(mpv.conf)与主仓库合并更新呢…结果还是覆盖更新/手动合并

**Reply by moomiji** [reply]
哎整成那样可得花时间了，而且主仓库更新后怎么解决冲突，自动解决冲突后的配置是否是想要的，也没想到好的方案（git subtree是个方向，但过于无感了，时间一久容易忘记使用过subtree）。
毕竟是不满当前的更新方案，摸个脚本减轻下之后需要更新时的负担，时间成本能接受，我也乐于把我的更新方案分享出来。
